src/main.rs

```diff
@@ -13,11 +16,22 @@ use crate::clients::tiled_client::TiledClient;
 use crate::config::GlazedConfig;
 use crate::handlers::graphql::graphql_handler;
 use crate::schemas::TiledQuery;
+use cli::{Cli, Commands};
 
 #[tokio::main]
 async fn main() -> Result<(), Box<dyn error::Error>> {
-    let config = GlazedConfig::from_file("config.toml")?;
+    let cli = Cli::parse();
 
+    let config_filepath = cli.config_filepath.unwrap_or(PathBuf::from("config.toml"));
```

Comment:
(nit as this one might be a style/personal preference thing but) using `.into()` instead of `T::from` is shorter and means you aren't as tied to the type if things get refactored later.

```suggestion
    let config_filepath = cli.config_filepath.unwrap_or("config.toml".into());
```